app/models/ems_event/automate.rb

```diff
@@ -12,6 +12,18 @@ def refresh(*targets, sync)
       EmsRefresh.queue_refresh(refresh_targets, nil, sync)
     end
 
+    def refresh_new_target
+      ems = ext_management_system
+      if ems.supports_refresh_new_target?
```

Comment:
There is a `SupportsFeatureMixin` that handles this type of check
Not sure this belongs here, might be better to have `EmsRefresh` handle what type of refresh a provider can handle.